init

```diff
@@ -9,10 +9,35 @@ exec &> >(tee "debug.log")
 
 ### Functions
 
+ask () {
+    read -p "> $1 " -r
+    echo
+}
+
 print () {
     echo -e "\n\033[1m> $1\033[0m"
 }
 
+download_and_execute_arch-config () {
+    ask "Do you want to download and execute arch-config?"
+    if [[ $REPLY =~ ^[Yy]$ ]]
+    then
+
+        if [[ ! -f /usr/bin/git ]]
+        then
+
+            pacman -S git --noconfirm &>/dev/null
+
+        fi
+
+        git clone https://github.com/stevleibelt/arch-config
```

Comment:
This links to your arch-config project. Other users don't want this, neither mine.